collector/src/main/java/com/bitmovin/analytics/data/EventData.kt

```diff
@@ -69,7 +70,7 @@ class EventData(
     var progUrl: String? = null
     var isMuted = false
     var sequenceNumber: Int = 0
-    val platform: String = if (deviceInfo.isTV) "androidTV" else "android"
+    var platform: String? = null
```

Comment:
Let's please not make this nullable! This just leads to someone forgetting some hook somewhere and we have null being sent to the server :/

Reply:
Considering https://github.com/bitmovin/bitmovin-analytics-collector-android/pull/130#discussion_r533278132, do you see a quick/nice way of working around this?